src/Build/Definition/Project.cs

```diff
@@ -855,27 +855,27 @@ public static string GetEvaluatedItemIncludeEscaped(ProjectItemDefinition item)
         /// <example>
         ///
         /// <code>
```

Comment:
📝 This `<code>` element appears to be missing a `<![CDATA[...]]>` wrapper around its content.